app/src/main/java/com/example/drawingactivity/DialogUtils.kt

```diff
@@ -149,7 +187,7 @@ object DialogUtils {
          * Lambda that provides the string of the current color.
          * Returns nothing. Used to confirm the use of the color.
          */
-        onSubmit: ((Hsv?) -> Unit)
+        onSubmit: (Hsv?) -> Unit
```

Comment:
In the future I will require your cleanup changes to be in a different PR. At work people will just not review and ask you to separate

Reply:
Understood, will not make changes in a ticket PR again and will create a separate commit when I chose to do these things.